packages/astro-ghostcms/src/integrations/rssfeed/astro-ghostcms-rss.ts

```diff
@@ -0,0 +1,54 @@
+import { createResolver, defineIntegration } from "astro-integration-kit";
+import { corePlugins } from "astro-integration-kit/plugins";
+import { z } from "astro/zod";
+import c from "picocolors";
+
+export default defineIntegration({
+	name: "@matthiesenxyz/astro-ghostcms-rss",
+	optionsSchema: z.object({
+		verbose: z.coerce.boolean().optional(),
+	}),
+	plugins: [...corePlugins],
+	setup({ options }) {
+		const { resolve } = createResolver(import.meta.url);
+
+		return {
+			"astro:config:setup": ({ watchIntegration, injectRoute, logger }) => {
+				watchIntegration(resolve());
+				const RSSLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
+						"RSSGenerator",
+					)}`,
+				);
+
+				RSSLogger.info(c.bold(c.magenta("RSS Feed Enabled. Setting up...")));
+
+				const rssRoute = "@matthiesenxyz/astro-ghostcms/rss-routes";
+
+				injectRoute({
+					pattern: "/rss-style.xsl",
+					entrypoint: `${rssRoute}/rss-style.xsl.ts`,
```

Comment:
You can probably resolve these paths instead of exporting it from your package?

Reply:
They did in the old days before `createResolver` ❤️ it just can't be a virtual module

Reply:
Yeah.... Thats updated now across the board... and those exports for `/rss-routes` and `/open-graph` are now moved to `resolve()`